.github/workflows/build.yml

```diff
@@ -40,16 +56,24 @@ jobs:
 
       - name: Upgrade python dependencies
         run: |
-          pip install --upgrade pip setuptools wheel
+          python -m pip install --upgrade pip
           pip --version
+          pip install --upgrade setuptools wheel twine check-wheel-contents
 
       - name: Build distributions
         run: ./scripts/build-dist.sh
 
       - name: Show distributions
-        shell: bash
         run: ls -lh dist/
 
+      - name: Check distribution descriptions
+        run: |
+          twine check dist/*
+
+      - name: Check wheel contents
+        run: |
+          check-wheel-contents dist/*.whl --ignore W007,W008
+
```

Comment:
moved this up, since this only needs to be checked once, not for different OSs

Reply:
also, if these checks fail then it makes sense to fail early and not try to verify installation at all